report/unsafe/sections/albin_arrowhead_communication.tex

```diff
@@ -0,0 +1,92 @@
+\subsection{Arrowhead framework}
+Arrowhead framework is a framework designed to aid in the functonality of Iot-devices, havning a focus on large scale industry and
+productivity. The framework provides a number of core system, systems that are required to run to enable certain functionality. This 
+project uses five of them nameley, service registry, orchestrator, authorization, gateway and gatekeeper. The service registry is used 
+to register systems and services, a system is defined as either a provider or a consumer. A provider provides a service for examample 
```

Comment:
"to register systems and services," porb better with . instead of ,?
spelling error : examample